drop-storage/src/types.rs

```diff
@@ -179,6 +179,12 @@ pub enum Event {
         file_id: FileId,
         progress: i64,
     },
+    Reject {
+        transfer_type: TransferType,
+        transfer_id: TransferId,
+        file_id: FileId,
+        by_peer: bool,
+    },
```

Comment:
The naming is inconsistent. The same goes for `Progress`. I think adding `File` prefix would be nice here

Reply:
Done